LottoDiary/LottoDiary/Sources/Present/ViewController.swift

```diff
@@ -28,6 +36,14 @@ class ViewController: UIViewController {
             make.size.equalTo(200)
         }
         // Do any additional setup after loading the view.
+        
+        view.addSubview(codeReviewLabel)
+        codeReviewLabel.snp.makeConstraints { make in
+            make.top.equalTo(testLabel.snp.bottom).inset(20)
+            make.leading.trailing.equalToSuperview().offset(30)
+            make.width.equalToSuperview()
+            make.height.equalTo(50)
+        }
```

Comment:
```suggestion
        
        view.addSubview(codeReviewLabel)
        codeReviewLabel.snp.makeConstraints { make in
            make.top.equalTo(testLabel.snp.bottom).offset(20)
            make.leading.trailing.equalToSuperview().inset(30)
            make.width.equalToSuperview()
            make.height.equalTo(50)
        }
```
보통 offset은 다른 객체와의 비교에 사용하교 inset은 eqaultoSuperview와 사용합니다